coalib/output/ConfWriter.py

```diff
@@ -105,7 +105,7 @@ def __write_key_val(self, keys, val):
 
         self.__file.write((self.__key_delimiter + " ").join(keys) + " " +
                           self.__key_value_delimiter + " " + val + "\n")
-        self.__wrote_newline = False
+        self.__wrote_newline = True
```

Comment:
You mean the the line `self.__wrote_newline = True`. I want me to remove that right? Yep it not of any use except for one case. I will update the PR by removing the above line and update the tests.